stock_production_lot_active/views/stock_production_lot.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright 2020 ACSONE SA/NV
-     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
```

Comment:
Just curious, did pre-commit remove the copyright statement, or did you do that? I don't know if there ever was any agreement in the community about copyright statement on data files, but I would not remove them manually.

Reply:
Good question, can't remember. Should I restore the copyright statement?

Reply:
Yes, please give it a try. If it disappears again we'll know it's the pre-commit doings!